recipes/create-dmg/all/conanfile.py

```diff
@@ -13,34 +13,39 @@ class CreateDmgConan(ConanFile):
     topics = "command-line", "dmg"
     homepage = "https://github.com/create-dmg/create-dmg"
     url = "https://github.com/conan-io/conan-center-index"
-    settings = "os", "arch", "compiler", "build_type"
+    settings = "os"
```

Comment:
Sicne this is a tool, it should have all the settings here but delete them from package id

```suggestion
    settings = "os", "arch", "compiler", "build_type"
```